### PR TITLE
Add server hostname to path breadcrumbs

### PR DESCRIPTION
**Pull request by MahmoudHamdy02**
Part of #229 

This implementation can probably be improved. Also I'm not sure if the `dbus` code should be placed here or the the main `app.jsx` instead.

### CODE REVIEW

File: src/navigatorBreadcrumbs.jsx [hunk 1]
```diff
@@ -17,12 +17,37 @@
...
+
+    const hostnameText = () => {
+        if (!hostnameData)
+            return undefined;
+
+        const pretty_hostname = hostnameData.PrettyHostname;
+        const static_hostname = hostnameData.StaticHostname;
+        let str = hostnameData.Hostname;
+        if (pretty_hostname && static_hostname && static_hostname !== pretty_hostname)
+            str = pretty_hostname + " (" + static_hostname + ")";
```
Comment (cockpituous) [inline src/navigatorBreadcrumbs.jsx:44]: These 2 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-245-20240205-200740-ed73c12c-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: src/navigatorBreadcrumbs.jsx [hunk 2]
```diff
@@ -17,12 +17,37 @@
...
+        else if (static_hostname)
+            str = static_hostname;
+
+        return str || '';
```
Comment (cockpituous) [inline src/navigatorBreadcrumbs.jsx:48]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-245-20240205-200740-ed73c12c-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)

**Comment by allisonkarlitskaya** [comment]
please take a look at #234

**Comment by MahmoudHamdy02** [comment]
Oh, didn't notice the hostname change is included in that PR. Thanks

**Comment by allisonkarlitskaya** [comment]
Sorry for the clash.  Happy to see you around again!

**Pull request closed without merging**